Example/OriginateUI/CodeViewController.m

```diff
@@ -0,0 +1,42 @@
+//
+//  CodeViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 6/22/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "CodeViewController.h"
+@import OriginateUI;
+
+@interface CodeViewController ()
+
+@end
+
+@implementation CodeViewController
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    self.textView = [[UITextView alloc] initWithFrame:CGRectMake(5, (self.view.frame.size.height*0.25f), (self.view.frame.size.width-10.0f), (self.view.frame.size.height*0.15f))];
+    self.view.backgroundColor = [UIColor whiteColor];
+    self.automaticallyAdjustsScrollViewInsets = NO;
+    self.textView.delegate = self;
+    [self.textView setBackgroundColor: [UIColor hex:0xfaebd7]];
+    [self.textView setReturnKeyType:UIReturnKeyDone];
+    self.textView.text = self.myString;
+    [self.view addSubview:_textView];
```

Comment:
It's not ideal to use the instance variable of a property outside of the setter and getter of the property, and the initialization method of the class the property belongs to. Because in the objc world... we frequently do some manipulation for the property in the property's setter and getter by operating on its instance variable, such as lazy getter.